include/aa_permitted_pipelines_generic.hpp

```diff
@@ -850,13 +850,14 @@ namespace astroaccelerate {
 
 			timer.Stop();
 			time_log.adding("FDAS", "total", timer.Elapsed());
+			time_log.adding("Total", "total", timer.Elapsed());
 			float time = timer.Elapsed()/1000;
 			printf("\n\n === OVERALL TDAS THROUGHPUT INCLUDING SYNCS AND DATA TRANSFERS ===\n");
```

Comment:
Its TDAS or FDA overall?

Reply:
This is for both. The way the pipeline is organized we must put Total - total after every module. That is why we have two invocations one is to add the time for FDAS and the other to add FDAS time to total time. Alternative approach would be to eliminate all "total" and just sum the execution time of all module as the total. But we have chosen this approach because sometimes the operation cost involve allocating and de-allocating memory which was/is bugged and it sometimes it takes a lot of time which would be missed otherwise.

Reply:
I thought so. Just to be sure than sorry